src/DelphiRest.inc

```diff
@@ -133,6 +133,46 @@
   {$DEFINE USE_NS}
 {$ENDIF VER290}
 
+{$IFDEF VER300}  // RAD Studio 10 Seattle
+  {$DEFINE DELPHI_7_UP}
+  {$DEFINE DELPHI_2007_UP}
+  {$DEFINE DELPHI_2009_UP}
+  {$DEFINE DELPHI_2010_UP}
+  {$DEFINE DELPHI_XE_UP}
+  {$DEFINE DELPHI_XE2_UP}
+  {$DEFINE DELPHI_XE3_UP}
+  {$DEFINE DELPHI_XE4_UP}
+  {$DEFINE DELPHI_XE5_UP}
+  {$DEFINE DELPHI_XE6_UP}
+  {$DEFINE DELPHI_XE7_UP}
+  {$DEFINE DELPHI_XE8_UP}
+  {$DEFINE USE_NS}
+  {$DEFINE DELPHI_10SEATTLE}
+  {$DEFINE DELPHI_10SEATTLE_UP}
+{$ENDIF}
+
+
+{ Provide compatibility for the next version }
+{$IFDEF CONDITIONALEXPRESSIONS} // Delphi 6+
+  {$IF CompilerVersion >= 30.0} // Delphi 10 Seattle
+    {$DEFINE DELPHI_7_UP}
+    {$DEFINE DELPHI_2007_UP}
+    {$DEFINE DELPHI_2009_UP}
+    {$DEFINE DELPHI_2010_UP}
+    {$DEFINE DELPHI_XE_UP}
+    {$DEFINE DELPHI_XE2_UP}
+    {$DEFINE DELPHI_XE3_UP}
+    {$DEFINE DELPHI_XE4_UP}
+    {$DEFINE DELPHI_XE5_UP}
+    {$DEFINE DELPHI_XE6_UP}
+    {$DEFINE DELPHI_XE7_UP}
+    {$DEFINE DELPHI_XE8_UP}
+    {$DEFINE USE_NS}
+    {$DEFINE DELPHI_10SEATTLE_UP}
+  {$IFEND}
+{$ENDIF}
+
```

Comment:
Yeah, I understood that. But, the Delphi 10 is included in this second block (CompilerVersion >= 30.0), right? 

The only difference between the two blocks is the `{$DEFINE DELPHI_10SEATTLE}` directive. I don't see this directive being used anywhere, so, why not mantain only the second block?